src/continuation/continuation_featurizing_find_and_match.jl

```diff
@@ -0,0 +1,134 @@
+export FeaturizingFindAndMatch, FFAM
+import ProgressMeter
+using Random: MersenneTwister
+
+"""
+    FeaturizingFindAndMatch <: AttractorsBasinsContinuation
+    FeaturizingFindAndMatch(mapper::AttractorsViaFeaturizing; kwargs...)
+
+A method for [`continuation`](@ref) as in [Datseris2023](@cite) that uses the featurizing algorithm for finding attractors ([`AttractorsViaFeaturizing`](@ref))
+and the "matching attractors" functionality offered by [`match_continuation!`](@ref). Based heavily on the `RecurrencesFindAndMatch`, which uses the recurrences algorithm ([`AttractorsViaRecurrences`](@ref)).
+
+You can use `FFAM` as an alias.
+
+## Keyword arguments
+
+- `distance = Centroid(), threshold = Inf, use_vanished = !isinf(threshold)`:
+  propagated to [`match_continuation!`](@ref).
+- `info_extraction = identity`: A function that takes as an input an attractor (`StateSpaceSet`)
+  and outputs whatever information should be stored. It is used to return the
+  `attractors_info` in [`continuation`](@ref). Note that the same attractors that
+  are stored in `attractors_info` are also used to perform the matching in
+  [`match_continuation!`](@ref), hence this keyword should be altered with care.
+- `seeds_from_attractor`: A function that takes as an input an attractor and the number of
+  initial conditions to sample from the attractor, and returns a vector with the sampled initial
+  conditions. By default, we just select the initial conditions randomly from the attractor.
+
+## Description
+
+At the first parameter slice of the continuation process, attractors and their fractions
+are found as described in the [`AttractorsViaFeaturizing`](@ref) mapper identifying groups
+of features. From each group of features, an attractor is extracted by sampling one point
+in the group, evolving it forwards in time, and collecting the trajectory. That is why
```

Comment:
What is the sampling exactly? Is it one of the initial conditions belonging to a particular feature group?